test/controllers/sessions_controller_test.rb

```diff
@@ -11,14 +11,14 @@ class SessionsControllerTest < ActionController::TestCase
     ShopifyApp.configuration.embedded_app = true
     auth_url = '/auth/shopify?shop=my-shop.myshopify.com'
     get :new, shop: 'my-shop'
-    assert response.body.match(/window\.top\.location\.href = "#{Regexp.escape(auth_url)}"/)
+    assert_match /window\.top\.location\.href = "#{Regexp.escape(auth_url)}"/, response.body
```

Comment:
could you `assert_match` on the `meta http-equiv` as well? I'm worried about the url quoting not working properly ?